pandas/core/frame.py

```diff
@@ -6045,7 +6045,7 @@ def _gotitem(self,
                  ):
         # type: (...) -> Union[Series, DataFrame]
         """
-        sub-classes to define
+        Sub-classes to define
         return a sliced object
```

Comment:
Can you use `Sub-classes to define. Return a sliced object.`

Or something better if you want.